### DISCUSSION TITLE
Welcome to elia Discussions!

### DISCUSSION THREAD

**Discussion by darrenburns** [darrenburns/elia#11] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mesutguzel** [comment]
I was trying to save some bash file into my local. I guess it is not possible for now. But would you help me to do that? That would be game changer.
